### ISSUE TITLE
Unintentional loop about calc.pandas

### ISSUE THREAD

**Issue by hori1537** [mordred-descriptor/mordred#78]
description

--
 test.py
 import pandas as pd
 from mordred import descriptors, Calculator
 from rdkit import  Chem

 i = 0 
 print('program start')
 i += 1
 print(i)

def fit_by_mordred():
    print('start the def_xxxx')
    df_mordred=pd.DataFrame({   'smiles' : ['C', 'CC', 'CCC'],
                                'name'   : ['a', 'b', 'C'],
                                'value'  : [1,2,3,] })

    df_mordred=pd.DataFrame({   'smiles' : ['C'],
                                'name'   : ['a'],
                                'value'  : [1] })

    def apply_molfromsmiles(smiles_name):
        mols = Chem.MolFromSmiles(smiles_name)
        return mols

    df_mordred['mols'] = df_mordred['smiles'].map(apply_molfromsmiles)
    #print(df_mordred)

    calc =Calculator(descriptors, ignore_3D = True)
    print('before calc.pandas')
    X = calc.pandas(df_mordred['mols']).astype('float').dropna(axis = 1)
    print('after calc.pandas')

if __name__ == "__main__":
    fit_by_mordred()

--

--
results
program start
1
start the def_xxxx
before calc.pandas
program start
1
program start
1
program start
1
program start
1
  0%|                                                      | 0/1 [00:00<?, ?it/s]program start
1
100%|██████████████████████████████████████████████| 1/1 [00:00<00:00,  4.10it/s]
after calc.pandas

--

environment
--
### OS/distribution

Windows 10

### conda or pip

pip

### python version

Please execute the command and paste result.

python 3.7.4

### library version

Please execute the command and paste result.

* pip

mordred              1.2.0
tensorflow           2.0.0

* conda
absl-py                   0.8.0                     <pip>
astor                     0.8.0                     <pip>
backcall                  0.1.0                     <pip>
blas                      1.0                         mkl
ca-certificates           2019.8.28                     0
cairo                     1.14.12              hf171d8a_3
category-encoders         2.1.0                     <pip>
certifi                   2019.9.11                py37_0
chainer                   6.4.0                     <pip>
chainer-chemistry         0.6.0                     <pip>
colorama                  0.4.1                     <pip>
cycler                    0.10.0                    <pip>
decorator                 4.4.0                     <pip>
filelock                  3.0.12                    <pip>
freetype                  2.9.1                ha9979f8_1
gast                      0.2.2                     <pip>
google-pasta              0.1.7                     <pip>
grpcio                    1.24.1                    <pip>
h5py                      2.10.0                    <pip>
icc_rt                    2019.0.0             h0cc432a_1
icu                       58.2                 ha66f8fd_1
intel-openmp              2019.4                      245
ipython                   7.8.0                     <pip>
ipython-genutils          0.2.0                     <pip>
jedi                      0.15.1                    <pip>
joblib                    0.14.0                    <pip>
jpeg                      9b                   hb83a4c4_2
Keras                     2.3.1                     <pip>
Keras-Applications        1.0.8                     <pip>
Keras-Preprocessing       1.1.0                     <pip>
kiwisolver                1.1.0                     <pip>
libboost                  1.67.0               hd9e427e_4
libpng                    1.6.37               h2a8f88b_0
libtiff                   4.0.10               hb898794_2
Markdown                  3.1.1                     <pip>
matplotlib                3.1.1                     <pip>
mkl                       2019.4                      245
mkl-service               2.3.0            py37hb782905_0
mkl_fft                   1.0.14           py37h14836fe_0
mkl_random                1.1.0            py37h675688f_0
mordred                   1.2.0                     <pip>
networkx                  2.3                       <pip>
numpy                     1.16.5           py37h19fb1c0_0
numpy-base                1.16.5           py37hc3f5095_0
olefile                   0.46                     py37_0
openssl                   1.1.1d               he774522_2
opt-einsum                3.1.0                     <pip>
pandas                    0.25.1           py37ha925a31_0
parso                     0.5.1                     <pip>
patsy                     0.5.1                     <pip>
pickleshare               0.7.5                     <pip>
pillow                    6.2.0            py37hdc69c19_0
pip                       19.2.3                   py37_0
pixman                    0.38.0               he774522_0
prompt-toolkit            2.0.10                    <pip>
protobuf                  3.7.1                     <pip>
PubChemPy                 1.0.4                     <pip>
py-boost                  1.67.0           py37h8300f20_4
Pygments                  2.4.2                     <pip>
pyparsing                 2.4.2                     <pip>
python                    3.7.4                h5263a28_0
python-dateutil           2.8.0                    py37_0
pytz                      2019.3                     py_0
PyYAML                    5.1.2                     <pip>
rdkit                     2019.03.4.0      py37hc1e3ae0_1    rdkit
scikit-learn              0.21.3                    <pip>
scipy                     1.3.1                     <pip>
setuptools                41.4.0                   py37_0
six                       1.12.0                   py37_0
sqlite                    3.30.0               he774522_0
statsmodels               0.10.1                    <pip>
tensorboard               2.0.0                     <pip>
tensorflow                2.0.0                     <pip>
tensorflow-estimator      2.0.0                     <pip>
termcolor                 1.1.0                     <pip>
tk                        8.6.8                hfa6e2cd_0
tqdm                      4.36.1                    <pip>
traitlets                 4.3.3                     <pip>
typing                    3.7.4.1                   <pip>
typing-extensions         3.6.6                     <pip>
vc                        14.1                 h0510ff6_4
vs2015_runtime            14.16.27012          hf0eaf9b_0
wcwidth                   0.1.7                     <pip>
Werkzeug                  0.16.0                    <pip>
wheel                     0.33.6                   py37_0
wincertstore              0.2                      py37_0
wrapt                     1.11.2                    <pip>
xz                        5.2.4                h2fa13f4_4
zlib                      1.2.11               h62dcd97_3
zstd                      1.3.7                h508b16e_0
